external/fv3fit/tests/emulation/layers/test_fields.py

```diff
@@ -97,20 +106,6 @@ def test_IncrementedFieldOutput(dt_sec: float):
     assert magnitude == pytest.approx(dt_sec, rel=1.0)
 
 
-def test_IncrementedFieldOutput_tendency_layer_name():
```

Comment:
I could probably revert this change...but I think we should probably remove the IncremntedFieldOutput object anyway, since we use `tensor_transforms` for that.